mailing_christmas/models/partner_compassion.py

```diff
@@ -29,3 +31,13 @@ class ResPartner(models.Model):
     def compute_uuid(self):
         for partner in self:
             partner.uuid = uuid.uuid4()
+
+    @api.depends('ref')
+    @api.multi
+    def compute_pays_christmas_fund(self):
+        self.pays_christmas_fund = False
+        for contract_line in self.other_contract_ids.contract_line_ids:
```

Comment:
add `.. in self.other_contract_ids.contract_line_ids.with_context(lang='en_US')` to be sure to have the names in English. Otherwise if Odoo user has French language, the name template will return "Fonds de noel".